apis/sgv2-docsapi/mvnw.cmd

```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```

Comment:
this one as well, delete

Reply:
Done in https://github.com/stargate/stargate/pull/1965/commits/57561e6fcbfaf8b6c9252fb55dc8cdedd1bcbf81